client/ayon_core/hosts/houdini/plugins/publish/collect_arnold_rop.py

```diff
@@ -85,6 +86,14 @@ def process(self, instance):
             files_by_aov[label] = self.generate_expected_files(instance,
                                                                aov_product)
 
+            # Set to False as soon as we have a separated aov.
+            multipartExr = False
```

Comment:
I think this logic is invalid. It's totally fine for an arnold render to have AOVs but write them into a single EXR as multilayers - currently this logic looks like; If any AOVs are enabled then `multipart=False` which seems invalid logic?

Reply:
If any AOVs are enabled and _rendered separately_ then then `multipart=False` 
ROPs have AOVs options but render them as one EXR file. 

I don't know if there's an option in arnold to specify to render as one layer or not but the logic in arnold collector always consider AOVs as separated products, so I assumed they are rendered in separated files. 

https://github.com/ynput/ayon-core/blob/f64bb5c1ea6182b3b3ec251f8aaf3785ddeda38c/client/ayon_core/hosts/houdini/plugins/publish/collect_arnold_rop.py#L71-L86

_For information: I didn't test arnold but I followed the same as mantra and Karma_

Reply:
I see what you mean.

I think the logic here for Arnold is wrong in that it assumes that _when_ there are AOVs that it must be non-merged / non-multilayer since it's totally valid to render multiple AOVs into a single file - which is exactly what multipart exr is for?

Not sure how the multipart/multilayer logic works in Houdini for AOVs but the Maya render logic does that fine where arnold with aovs can render to both multilayer or separate products.

Reply:
> how the multipart/multilayer logic works in Houdini for AOVs 

As far as I tested render rops in Houdini are not consisted. 
Some has `multipart` check box explicitly and some add the check box per AOV.
So, to align with the current logic in deadline plugin I had to deduce the `multipart` value.
In a nutshell the logic is as follow: 
```
if `review` key in instance data is enabled: 
    if the render is multipart Exr (One merged exr) then add review tag.
    if not then check against AOV regex filter in settings, if matches the regex then add review tag.
```

Reply:
I added a todo about it, therefore I'll mark this as resolved. 
Feel free to open an issue thou,